concepticondata/retired.json

```diff
@@ -639,6 +639,11 @@
       "id": "Galicio-2015-28-28",
       "comment": "corrected typo",
       "replacement": "Galucio-2015-28-28"
+    },
+    {
```

Comment:
You'll have to add replacments for each of the entries, I'm afraid, as the 2016-125 is now the new ID :(